src/test/java/hudson/plugins/active_directory/docker/TheFlintstonesTest.java

```diff
@@ -98,6 +105,8 @@ public void setUp() throws Exception {
             }
             i ++;
         }
+        DOCKER_IP = d.ipBound(3268);
+        DOCKER_PORT = d.port(3268);
```

Comment:
Should you not move these lines up to the top of the method and then use the variables in the `ActiveDirectoryDomain` constructor?